### PR TITLE
Handle port correctly

### PR DESCRIPTION
**Pull request by eccles**
Allow the user to specify the name of the port env variable. Some services do not use the PORT variable when the do not have a grpcserver.

Add local logger to tracing

[AB#8760](https://dev.azure.com/jitsuin/0629f48c-3979-4bbc-9026-cb06b3dfd0ae/_workitems/edit/8760)

### CODE REVIEW

File: azblob/list.go [hunk 1]
```diff
@@ -16,12 +16,22 @@ func (azp *Storer) Count(ctx context.Context, tagsFilter string, opts ...Option)
...
 	logger.Sugar.Debugf("Count")
 
-	r, err := azp.FilteredList(ctx, tagsFilter, opts...)
-	if err != nil {
-		return 0, err
-	}
-
-	return int64(len(r.Items)), nil
+	var count int64
+	m := "counter"
```
Comment (honourfish) [inline azblob/list.go:20]: do we need to make our own marker, the marker is set after during the call to FilteredList, as the nextList marker?

File: azblob/list.go [hunk 2]
```diff
@@ -16,12 +16,22 @@ func (azp *Storer) Count(ctx context.Context, tagsFilter string, opts ...Option)
 
 	logger.Sugar.Debugf("Count")
 
-	r, err := azp.FilteredList(ctx, tagsFilter, opts...)
-	if err != nil {
-		return 0, err
-	}
-
-	return int64(len(r.Items)), nil
+	var count int64
```
Comment (robinbryce) [inline azblob/list.go:19]: var marker string

is idiomatic and less confusing

File: azblob/list.go (hunk 1 above)
Comment (eccles) [reply azblob/list.go]: please ignore this - its wrong

File: azblob/list.go (hunk 2 above)
Comment (eccles) [reply azblob/list.go]: yes - correct code is in another PR

**Review by j-hartley — APPROVED** [review]

**Pull request merged**